src/Inventory.js

```diff
@@ -6,17 +6,15 @@ class Inventory extends Map {
    */
   addItem(item) {
     this.set(item.uuid, item);
-    item.isHeld = true;
-    item.container = this;
+    item.belongsTo = this;
```

Comment:
You don't. Both Item and Character have their own addItem/removeItem. You shouldn't ever need to directly access the `inventory` of an entity.